appvalidator/specs/webapps.py

```diff
@@ -306,8 +306,7 @@ class WebappSpec(Spec):
                     "navigation": {"expected_type": bool},
                 }
             },
-            # For now, we only support languages pack targeting gaia apps and
-            # only with 2.2 version.
+            # For now, we only support languages pack targeting gaia apps
```

Comment:
nit: period at the end of the sentence. (also, maybe s/languages pack/language packs/ while you are at it ? it seems better)